owa/app/js/components/common/header.jsx

```diff
@@ -91,28 +91,27 @@ export default class Header extends React.Component {
           <NavbarBrand href="../../"><img src={image.src} alt={image.alt}/></NavbarBrand>
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav className="ml-auto" navbar>
-              <NavItem>
-                  <UncontrolledNavDropdown>
-                      <DropdownToggle nav caret> <FaUser/>
-                       {' ' + this.state.currentUser}
-                      </DropdownToggle>
-                      <DropdownMenu>
-                        <DropdownItem header><a href="#">My Account</a></DropdownItem>
-                      </DropdownMenu>
-                    </UncontrolledNavDropdown>
-              </NavItem>
-              <NavItem>
-                  <UncontrolledNavDropdown>
-                      <DropdownToggle nav caret> <FaMapMarker/>
-                       {(this.state.currentLocationTag != "")
-                          ? this.state.currentLocationTag
-                          : this.state.defaultLocation}
-                      </DropdownToggle>
-                      <DropdownMenu>
-                        {this.state.locationTags.map((location, index )=><DropdownItem key={index} header><a href="#" onClick={this.handleOnClick.bind(this, location.display)}>{location.display}</a></DropdownItem>)}
-                      </DropdownMenu>
-                    </UncontrolledNavDropdown>
-              </NavItem>
+              
+            <UncontrolledNavDropdown>
+                <DropdownToggle nav caret> <FaUser/>
+                  {' ' + this.state.currentUser}
+                </DropdownToggle>
+                <DropdownMenu>
+                  <DropdownItem header><a href="#">My Account</a></DropdownItem>
+                </DropdownMenu>
+              </UncontrolledNavDropdown>
+        
+            <UncontrolledNavDropdown>
+                <DropdownToggle nav caret> <FaMapMarker/>
+                  {(this.state.currentLocationTag != "")
+                    ? this.state.currentLocationTag
+                    : this.state.defaultLocation}
+                </DropdownToggle>
+                <DropdownMenu>
+                  {this.state.locationTags.map((location, index )=><DropdownItem key={index} header><a href="#" onClick={this.handleOnClick.bind(this, location.display)}>{location.display}</a></DropdownItem>)}
```

Comment:
It's more advisable to bind methods in the constructor. So here, you can simply do something like `onClick={() => this.handleOnClick(location.display)}`

Reply:
true

Reply:
@adamsdenniskariuki  I'll advise you to un-commit this file because I'm currently working on it and it does not involve this PR's scope

Reply:
@kwahalf The errors on the header file affect all tabs. It would be redundant to create a new PR for the issue which will be fixed with this PR.

Reply:
updated the file